### PR TITLE
Transfer old pages

### PR DESCRIPTION
**Pull request by IlyaKirsanov**

### CODE REVIEW

File: src/components/Header/Header.scss [hunk 1]
```diff
@@ -63,7 +63,7 @@
 
 .header.sticky{
     height: 80px;
-    box-shadow: 0 1px 10px 0 rgba(0, 0, 0, 0.1);
+    //box-shadow: 0 1px 10px 0 rgba(0, 0, 0, 0.1);
```
Comment (sergiylegmelnik) [inline src/components/Header/Header.scss:66]: Remove commented code

File: src/images&video/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+ import videoNeuralNetwork from '@assets/video/neural-network.mov'
+// import blackArrow from '@assets/img/right-arrow-black.svg'
+// import kubernetes from '@assets/img/kubernetes.svg'
+// import javascriptImg from '@assets/img/javascript.svg'
+// import angular from '@assets/img/angular.svg'
```
Comment (sergiylegmelnik) [inline src/images&video/index.js:5]: Remove all commented code

File: src/pages/CareersVacancies/Vacancy/VacancyDescription/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+export default class VacancyDescription extends React.PureComponent {
+    constructor(props) {
+        super(props)
+        this.state = { vacationId: currentRoute.context.params.id }
+        currentRoute.on(this.handleCustomEvent)
+    }
+
+    componentDidMount () {
+        // if(!vacations[this.state.vacationId]) router.page('#!/careers')
```
Comment (sergiylegmelnik) [inline src/pages/CareersVacancies/Vacancy/VacancyDescription/index.js:17]: Remove commented code

Comment (IlyaKirsanov) [reply src/pages/CareersVacancies/Vacancy/VacancyDescription/index.js]: done

File: src/images&video/index.js (hunk 2 above)
Comment (IlyaKirsanov) [reply src/images&video/index.js]: done

File: src/components/Header/Header.scss (hunk 1 above)
Comment (IlyaKirsanov) [reply src/components/Header/Header.scss]: done

**Pull request merged**